src/garage/envs/env_spec.py

```diff
@@ -54,3 +59,24 @@ def observation_space(self, observation_space):
 
         """
         self._output_space = observation_space
+
+    @property
+    def max_episode_steps(self):
+        """Get max episode steps.
+
+        Returns:
+            int: The maximum number of steps that an episode
+
+        """
+        return self.max_episode_steps
+
+    @max_episode_steps.setter
```

Comment:
please don't allow for setters. These should be immutable. We have #1239 open for the other setters in this file (we can probably remove them easily after you're done with this new API).